specs/protocol/identifiers.md

```diff
@@ -25,7 +25,7 @@ Is represented as an _outpoint_: a pair of [transaction ID](#transaction-id) as
 
 ### Deposit ID
 
-The UTXO ID of a deposit is computed as the [hash](./cryptographic_primitives.md#hashing) of TODO.
+The UTXO ID of a deposit is computed as the [hash](./cryptographic_primitives.md#hashing) of the recipient address `byte[32]`, token address `byte[32]`, token precision `uint8` and deposit nonce `uint64`: `hash(byte[32] ++ byte[32] ++ uint8 ++ uint64)`. Address values are serialized as a byte array of length 32 left-padded with zeroes, and all other value types are serialized according to the standard [transaction serialization](./tx_format.md#transaction).
```

Comment:
Just reposting this here so that it is not lost in the comments: Deposits will not be able to be used until they are finalized, I wouldn't recommend SDK to include it inside the next tx until we are sure that finalization is done.
We can make that contract sends us nonce as id, while we mutate this into the id (hash of all data) that we want to be presented to users/sdk?